Truck.js

```diff
@@ -1,6 +1,27 @@
 var Car = require('./Car.js');
 
 //Implement your Truck here
-function Truck() {};
+
+function Truck(make, model, year, color, seats, capacity, cargo) {
+	Car.call(this, make, model, year, color);
+	this.seats = 3;
+	this.capacity = capacity || 0;
+	this.cargo = cargo || 0;
+};
+
+Truck.prototype = Object.create(Car.prototype);
+Truck.prototype.constructor = Truck;
+
+Truck.prototype.load = function(pounds) {
+	this.cargo = pounds;
+	if (this.cargo < this.capacity) {
+		return true;
+		pounds++;
+	} else {
+		return false;
+	}
+}
+
+
```

Comment:
Here are the solutions for the ```Truck.prototype.load``` method and the Truck.prototype.unload``` method:
```
Truck.prototype.load = function(amount) {
	if (amount + this.cargo <= this.capacity) {
		this.cargo += amount;
		return true;
	}
	return false;
}

Truck.prototype.unload = function(amount) {
	if (amount <= this.cargo) {
		this.cargo -= amount;
		return true;
	}
	return false;
}
```